app/javascript/dashboard/components/widgets/conversation/contextMenu/Index.vue

```diff
@@ -34,15 +34,13 @@
       :option="labelMenuConfig"
       :sub-menu-available="!!labels.length"
     >
-      <template>
```

Comment:
Was this intended?

Reply:
Yeah, based on [no-lone-template](https://eslint.vuejs.org/rules/no-lone-template.html) rule